cfme/tests/intelligence/reports/test_menus.py

```diff
@@ -132,17 +133,14 @@ def test_add_reports_to_available_reports_menu(
 @pytest.fixture()
 def rbac_user(appliance, request, group):
     user, user_data = _users(request, appliance, group=group)
-    yield appliance.collections.users.instantiate(
+    return appliance.collections.users.instantiate(
         name=user[0].name,
```

Comment:
Question:
Why did you change it to ```return```?

Reply:
Because I am not going to come back to this fixture for any cleanup, so I just return the values.